issue-1/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
You don't want your pages to be in the issue-1 folder. You just want index.html in the root of the project.